src/containers/ProductComponent.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+
+const ProductComponent = () => {
+  const products = useSelector((state) => state.allProducts.products);
+  const renderList = products.map((product) => {
+    const {
+      id, title, image, price, category,
+    } = product;
+    return (
+      <div className="eight wide column main_container" key={id}>
+        <Link to={`/product/${id}`}>
+          <div className="ui link cards">
+            <div className="card">
```

Comment:
- [ ] I think you forgot to implemet the filter functionality here so I recommend to make it since it is one of the requirements 💯🚀🔥 so kindly implement it  so you can use search and use filter to the data 🔥

![image](https://user-images.githubusercontent.com/58553711/203681989-20db62d8-f031-4664-9b7d-5746fb024b0d.png)

Reply:
- Hey, as requested by the previous reviewer, please kindly address the issue stated above as this would improve the user experience by helping them search a particular product from your app. please, kindly fix this issue.

Reply:
- [ ] This issue is not yet resolved :x: